test/Combinatorics/Graph.jl

```diff
@@ -196,12 +196,18 @@
         label!(G1, nothing, vertex_labels; name=:color)
         @test_throws ArgumentError G1.color[1, 2]
         @test G1.color[1] == 2
+
+        edge_labels = Dict((1, 2) => 1, (3, 4) => 2)
+        label!(G1, edge_labels, nothing; name=:color)
+        @test G1.color[1, 2] == 1
```

Comment:
```suggestion
        @test G1.color[1, 2] == 1
        @test G1.color[1] == 2
```
To check that the vertex label is still there.